pysatNASA/tests/test_instruments.py

```diff
@@ -8,13 +8,24 @@
 
 import warnings
 
+import pysat
 import pytest
 
 # Make sure to import your instrument library here
 import pysatNASA
 
 # Import the test classes from pysat
 from pysat.tests.classes import cls_instrument_library as clslib
+from pysat.utils import testing
```

Comment:
All the pysat imports should be grouped together.  I recommend moving up pytest to the warnings block and moving all the pysat imports above the pysatNASA import.